src/templates/core/CancelablePromise.hbs

```diff
@@ -34,47 +33,47 @@ export class CancelablePromise<T> implements Promise<T> {
 			onCancel: OnCancel
 		) => void
 	) {
-		this.#isResolved = false;
-		this.#isRejected = false;
-		this.#isCancelled = false;
-		this.#cancelHandlers = [];
-		this.#promise = new Promise<T>((resolve, reject) => {
-			this.#resolve = resolve;
-			this.#reject = reject;
+		this.isResolved = false;
```

Comment:
I think the convention is to prefix with underscore for "private" fields, can you do that?